seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -4290,16 +4290,16 @@
     },
     {
       "id": "56533eb9ac21ba0edf2244c9",
-      "title": "Accessing Objects Properties with Variables",
+      "title": "Accessing Object Properties with Variables",
       "description": [
         "Another use of bracket notation on objects is to use a variable to access a property. This can be very useful for iterating through lists of the object properties or for doing the lookup.",
         "Here is an example of using a variable to access a property:",
         "<blockquote>var someProp = \"propName\";<br>var myObj = {<br>  propName: \"Some Value\"<br >}<br>myObj[someProp]; // \"Some Value\"</blockquote>",
         "Here is one more:",
         "<blockquote>var myDog = \"Hunter\";<br>var dogs = {<br>  Fido: \"Mutt\",\n  Hunter: \"Doberman\",\n  Snoopie: \"Beagle\"<br >}<br>var breed = dogs[myDog];<br>console.log(breed);// \"Doberman\"</blockquote>",
-        "Note that we  do <em>not</em> use quotes around the variable name when using it to access the property because we are using the <em>value</em> of the variable, not the <em>name</em>",
+        "Note that we  do <em>not</em> use quotes around the variable name when using it to access the property because we are using the <em>value</em> of the variable, not the <em>name</em>.",
         "<hr>",
-        "Use the <code>playerNumber</code> variable to lookup player <code>16</code> in <code>testObj</code> using bracket notation. Then assign that name to the <code>player</code> variable."
+        "Use the <code>playerNumber</code> variable to look up player <code>16</code> in <code>testObj</code> using bracket notation. Then assign that name to the <code>player</code> variable."
```

Comment:
@JoolsMcFly ah good point. Thanks for pointing that out. I neglected to notice "lookup" as a noun, not a verb. I wish [this site](http://www.notaverb.com) wasn't down, but [here's](https://web-beta.archive.org/web/20161102191934/http://notaverb.com/lookup) an internet archive of the site on a relevant subject 😄